backend/src/main/java/com/wire/bots/roman/MessageHandler.java

```diff
@@ -430,6 +435,24 @@ private boolean send(OutgoingMessage message) {
         }
     }
 
+    private boolean shouldIgnore(Provider provider, OutgoingMessage message) {
+        if (message.type.equalsIgnoreCase("conversation.bot_request"))
+            return false;
+
+        if (provider.commandPrefix == null)
+            return false;
+
+        if (provider.commandPrefix.equalsIgnoreCase("***"))
+            return true;
+
+        if (!message.type.equalsIgnoreCase("text"))
+            return true;
+
+        final String text = message.text.data;
+
+        return !text.startsWith(provider.commandPrefix);
```

Comment:
if text does not correspond to command prefix, it's ignored, is that correct? shouldn't it be other way around? that if it starts with command prefix, it's ignored?

Reply:
Yes. Unless the text does not start with that prefix the whole message is ignored (not forwarded to the bot)

Reply:
aaaah, makes sense now, sorry, I misinterpreted that!